packages/components/pie-thumbnail/src/defs.ts

```diff
@@ -1,3 +1,21 @@
-// TODO - please remove the eslint disable comment below when you add props to this interface
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface ThumbnailProps {}
+import type { ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
+
+export const variants = [
+    'default', 'outline',
+] as const;
+
+export type Variant = typeof variants[number];
+
+export interface ThumbnailProps {
+    variant?: Variant;
+    src?: string;
+    alt?: string;
+}
```

Comment:
we don't need to export type Variant here as users can access the interface items if they wish to be granular  such as ThumbnailProps['variant']

```suggestion

export interface ThumbnailProps {
    variant?: typeof variants[number];
    src?: string;
    alt?: string;
}
```